modules/userpaex/password.php

```diff
@@ -110,6 +110,7 @@
                         $oldPassword = '';
 
                         eZUser::setCurrentlyLoggedInUser( $user, $UserID );
+                        eZUser::updateLastVisit( $UserID );
```

Comment:
From the description it looks like this module logs you in. In that case it should probably update the login counter, too. Meaning, the second parameter should be set to true. If login doesn't happen here, then never mind.
https://github.com/ezsystems/ezpublish-legacy/blob/0d8aebb911ec2d8d4665c214797ae7629f134d7e/kernel/classes/datatypes/ezuser/ezuser.php#L1528
```suggestion
                        eZUser::updateLastVisit( $UserID, true );
```